shell/platform/linux/fl_dart_project.cc

```diff
@@ -19,6 +19,7 @@ struct _FlDartProject {
   gchar* aot_library_path;
   gchar* assets_path;
   gchar* icu_data_path;
+  GPtrArray* dart_entrypoint_args;
```

Comment:
It would probably be simpler to use `GStrv` and `g_strdupv`, `g_strv_length` etc rather than a pointer array.